ecosystem/platform/server/app/components/dialog_component.rb

```diff
@@ -12,7 +12,7 @@ class DialogComponent < ViewComponent::Base
   def initialize(**rest)
     @rest = rest
     @rest[:class] = [
-      'rounded-xl border-none bg-neutral-900 text-white p-8 w-96 fixed',
+      'rounded-xl border-none bg-neutral-900 text-white w-96 fixed p-0',
```

Comment:
Yes, since user agent stylesheet adds 16px padding to `<dialog>` by default which would then result in closing of the `<dialog>` when clicking within this padding as `event.target.nodeName` would return `'DIALOG'`.  

Alternative would be to set `padding:0` within `application.tailwind.css`.